config/routes.rb

```diff
@@ -3,6 +3,7 @@
 
   root 'static_pages#index'
   resources :games, only: %i[new create show index update] do
+    post 'forfeit', on: :member
```

Comment:
I'd go with a `put`. Because it is making a `change`, rather than creating something new.